src/public/downloads/Page.js

```diff
@@ -0,0 +1,324 @@
+      size: '5.1Gb',
+    },
+  },
+  {
+    version: '17.04',
+    date: '2017 Apr',
+    associations: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/17.04/17.04_association_data.json.gz',
+      size: '207MB',
+    },
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/17.04/17.04_evidence_data.json.gz',
+      size: '4.4Gb',
+    },
+  },
+  {
+    version: '17.02',
+    date: '2017 Feb',
+    associations: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/17.02/17.02_association_data.json.gz',
+      size: '215MB',
+    },
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/17.02/17.02_evidence_data.json.gz',
+      size: '4.35Gb',
+    },
+  },
+  {
+    version: '16.12',
+    date: '2016 Dec',
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.12/16.12_evidence_data.json.gz',
+      size: '4.35Gb',
+    },
+  },
+  {
+    version: '16.09',
+    date: '2016 Sep',
+    associations: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.04/16.08_association_data_fixed.json.gz',
+      size: '179MB',
+    },
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.08/16.08_evidence_data.json.gz',
+      size: '1.7Gb',
+    },
+  },
+  {
+    version: '16.08',
+    date: '2016 Aug',
+    associations: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.08/16.08_association_data.json.gz',
+      size: '179MB',
+    },
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.08/16.08_evidence_data.json.gz',
+      size: '1.7Gb',
+    },
+  },
+  {
+    version: '16.04',
+    date: '2016 Apr',
+    associations: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.04/16.04_association_data.json.gz',
+      size: '148MB',
+    },
+    evidence: {
+      url:
+        'https://storage.googleapis.com/open-targets-data-releases/16.04/16.04_evidence_data.json.gz',
+      size: '1.3Gb',
+    },
+  },
+];
+
+const DownloadsPage = () => {
+  return (
+    <Grid container>
+      <Grid item xs={12} md={6}>
+        <Typography variant="h5" component="h1">
+          Data Download
+        </Typography>
+        <br />
+        <Typography>
+          All data from targetvalidation.org is available for download as
+          compressed JSON files.
+          <br />
+          <br />
+          We provide downloads of all associations between targets and diseases
+          calculated by the platform, as well as all the evidence used in
+          calculating each association. These are the same objects returned by
+          the corresponding <tt>/public/associations</tt> and{' '}
```

Comment:
First time I've encountered this `<tt>`. Looks like it is obsolete now according to this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt. Also, HTML should be used for structure, not for styling.

Reply:
Good catch. Following suggestion on link, I've swapped `tt` tags for `code` tags.

Reply:
I also removed `br` tags and added the `paragraph={true}` prop on `Typography` components.